types/components/buttons.d.ts

```diff
@@ -108,6 +108,81 @@ export type OnShippingChangeActions = {
     };
 };
 
+type CurrencyCodeAndValue = {
+    currencyCode: string;
+    value: string;
+};
+
+type ShippingOption = {
+    amount: CurrencyCodeAndValue;
+    id?: string;
+    label: string;
+    selected: boolean;
+    type: "SHIPPING" | "PICKUP";
+};
```

Comment:
How would you feel about naming this `CheckoutShippingOption` to match the [equivalent type](https://github.com/paypal/paypal-smart-payment-buttons/blob/main/src/types.js#L381-L390) in SPB?